### PR TITLE
Enable TimeToMain on Linux

### PR DESCRIPTION
**Pull request by ooooolivia**

### CODE REVIEW

**Comment by ooooolivia** [comment]
From the log the fix to startup is working and TimeToMain parser is enabled:
https://helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-performance-refs-pull-1539-merge-2efc42a229eb480586/Startup%20-%20New%20Console%20Template%20-%20FDD%20Publish/console.1ef8ab4e.log?sv=2019-07-07&se=2020-10-22T01%3A05%3A37Z&sr=c&sp=rl&sig=0xBml%2FUKEvwY8rDBHHNEHk%2Ba0MONjdL6P5JFjZOOoII%3D

**Comment by ooooolivia** [comment]
NuGet.cofig problem fixed and all Linuax scenario workitems passed.  Ready to merge
https://dev.azure.com/dnceng/public/_build/results?buildId=839399&view=logs&j=d9f1e651-4f07-5793-5cbc-80aa7fd37b27&t=628acac2-6fff-560c-e951-ce74dae9c369

File: azure-pipelines.yml [hunk 1]
```diff
@@ -64,6 +64,22 @@ jobs:
...
+      kind: scenarios
+      architecture: x64
+      pool: Hosted Ubuntu 1604
+      queue: Ubuntu.1804.Amd64.Open
+      container: ubuntu_x64_build_container
+      projectFile: scenarios.proj
+      channels:
+        - master
+        - release/5.0.1xx
+        # - release/3.1.3xx
```
Comment (billwert) [inline azure-pipelines.yml:82]: Do we need to turn this on for physical machines also? This just the VM queue, yes?

**Review by billwert — CHANGES_REQUESTED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (ooooolivia) [reply azure-pipelines.yml]: Thanks for the reminder! added this in the new push.

**Pull request merged**